backend/satellite_tools/reposync.py

```diff
@@ -360,14 +360,32 @@ def clear_ssl_cache():
     shutil.rmtree(ssldir, True)
 
 
+def verify_certificates_dates(certs):
+    cert = ""
+    isCert = False
```

Comment:
Small nitpick: could we use a more pythonic `is_cert` name here instead of camelCase formatting?